app/javascript/controllers/interactions_controller.js

```diff
@@ -1,32 +1,49 @@
-import { Controller } from "@hotwired/stimulus"
+import { Controller } from '@hotwired/stimulus';
 
 export default class extends Controller {
-
   clicked(event) {
-    const targetElement = event.target
-    const isDescendantOfAddComment = event.target.closest('.add-comment') !== null;
-
-
-    if(isDescendantOfAddComment){
-      if(targetElement.classList.contains('comment-text'))
-        event.preventDefault()
-      event.stopPropagation()
-      return
-    } 
+    const targetElement = event.target;
+    const isDescendantOfAddComment =
+      event.target.closest('.add-comment') !== null;
+
+    console.log(
+      '---- targetElem -----',
```

Comment:
- [ ] For Javascript best practices, kindly ensure that you remove the `console.log()` from your code base as it’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.